docusaurus/docs-internals/editor-controls/data-controls/animation.md

```diff
@@ -49,40 +65,77 @@ Returns an object with the following values:
   style?: string;
 }
 
-EffectType = "none" | "bounce" | "fade" | "rotate" | "slide" | "zoom" | "attention" | "pulse" 
+EffectType = "none" | "bounce" | "fade" | "rotate"* | "rotate2"* | "slide" | "zoom" | "attention" | "pulse" 
   | "wobble" | "buzz" | "scale" | "skew" | "move"
 ```
 
-`type` - the type of the animation; <br/>
+`type` - the type of the animation (`"rotate"` is usually used on the elements when they enter the viewport, while `"rotate2"` is used to add a hover animation); <br/>
 `duration` - the duration of the animation in milliseconds; <br/>
 `delay` - the delay before the animation starts, in milliseconds; <br/>
 `infiniteAnimation` - indicates whether the animation should loop indefinitely; <br/>
 `big?` - specific to the `"fade"` animation, specifies if the animation will start outside the viewport; <br/>
 `direction?` - returned for all animations except `"attention"`, indicates the direction from which or to which the animation will be executed; <br/>
 `style?` - specific to the `"attention"` animation, variants of the `"attention"` animation; <br/>
 
+Example of the value: 
+
+```js
+{
+  delay: 0,
+  direction: "up",
+  duration: 600,
+  infiniteAnimation: true,
+  type: "move"
```

Comment:
as i see it return `name` instead of `type`

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/4a48c69c-e72a-4737-b5f3-c582c81e208c)

Reply:
idk why I was checking what returns `style`